pom.xml

```diff
@@ -59,7 +59,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <keycloak.version>${keycloak.version}</keycloak.version>
+        <keycloak.version>${keycloak.version}}</keycloak.version>
```

Comment:
${keycloak.version}} ? There are two `}`

Reply:
this is a mistake